packages/@aws-cdk/aws-codecommit/lib/repository.ts

```diff
@@ -110,8 +123,128 @@ export interface IRepository extends IResource {
    * Grant the given identity permissions to read this repository.
    */
   grantRead(grantee: iam.IGrantable): iam.Grant;
+
+
+  /**
+   *
+   * Defines a CodeStar Notification rule triggered when the project
+   * events specified by you are emitted. Similar to `onEvent` API.
+   *
+   * You can also use the methods to define rules for the specific event emitted.
+   * eg: `notifyOnPullRequstCreated`
+   *
+   * @param id
+   * @param target
+   * @param options
+   * @returns CodeStar Notifications rule associated with this repository
+   */
+  notifyOn(
+    id: string,
+    target: notifications.INotificationRuleTarget,
+    options: RepositoryNotifyOnOptions,
+  ): notifications.INotificationRule;
+
+  /**
+   * Defines a CodeStar Notification rule which triggers when a comment is made on a commit.
+   */
+  notifyOnCommentOnCommits(
+    id: string,
+    target: notifications.INotificationRuleTarget,
+    options?: notifications.NotificationRuleOptions,
+  ): notifications.INotificationRule;
+
+  /**
+   * Defines a CodeStar Notification rule which triggers when a comment is made on a pull request
+   */
+  notifyOnCommentOnPullRequests(
```

Comment:
```suggestion
  notifyOnPullRequestsComment(
```

Reply:
Apologies, this should actually be

```suggestion
  notifyOnPullRequestComment(
```